frontend/src/upload.js

```diff
@@ -99,6 +167,18 @@ $(document).ready(function() {
                    .then(str => {
                      notify(str);
                    });
+      localStorage.setItem('referrer', 'cancelled-upload');
+      
+      // record upload-stopped (cancelled) by sender
+      window.analytics
+            .sendEvent('sender', 'upload-stopped', {
+              cm1: file.size,
+              cm5: localStorage.getItem('totalUploads'),
+              cm6: unexpiredFiles,
+              cm7: totalDownloads,
+              cd1: event.type === 'drop' ? 'drop' : 'click',
```

Comment:
The only way for the `event.type` to not be `drop` is if the user doesn't drag and drop the file. I'm not sure the `onChange` property of the input tag can be changed with anything other than a click or a drag for a file upload.